tests/test-spawn-snid.lua

```diff
@@ -0,0 +1,44 @@
+--- Tests for spawn's startup notifications.
+
+local spawn = require("awful.spawn")
+
+local manage_called, c_snid
+
+client.connect_signal("manage", function(c)
+  manage_called = true
+  c_snid = c.startup_id
+  print("manage_called: c_snid:", tostring(c_snid))
```

Comment:
Oh and: Does this work for you on your own computer? As I said, it did work on mine. :-/